catalog-info.yaml

```diff
@@ -125,3 +125,50 @@ spec:
           access_level: MANAGE_BUILD_AND_READ
         everyone:
           access_level: READ_ONLY
+
+# ***********************************
+# SECTION START: E2E tests pipeline
+# ***********************************
+---
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: logstash-filter-elastic-integration-e2e
+  description: 'A pipeline for `elastic_integration` E2E tests.'
+  links:
+    - title: 'A pipeline for `elastic_integration` E2E tests.'
+      url: https://buildkite.com/elastic/logstash-filter-elastic-integration-e2e
+spec:
+  owner: group:ingest-fp
+  type: buildkite-pipeline
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      description: ':logstash: `elastic_integration` E2E tests. :pipeline:'
+      name: logstash-filter-elastic_integration-e2e
+    spec:
+      pipeline_file: .buildkite/e2e-pipeline.yml
+      maximum_timeout_in_minutes: 120
+      repository: elastic/logstash-filter-elastic_integration
+      env:
+        ELASTIC_PR_COMMENTS_ENABLED: 'true'
+        ELASTIC_SLACK_NOTIFICATIONS_ENABLED: 'true'
+        SLACK_NOTIFICATIONS_CHANNEL: '#logstash-build'
+        SLACK_NOTIFICATIONS_ON_SUCCESS: 'false'
+      schedules:
+        Weekly E2E Run:
+          branch: main
+          cronline: "@weekly"
+          message: "Run plugin E2E tests weekly."
+      teams:
+        logstash:
+          access_level: MANAGE_BUILD_AND_READ
+        ingest-fp:
+          access_level: MANAGE_BUILD_AND_READ
+        ingest-eng-prod:
+          access_level: MANAGE_BUILD_AND_READ
+        everyone:
+          access_level: READ_ONLY
```

Comment:
I suggest that we commit the changes to `catalog-info.yaml` in a separate PR (pointing to a dummy pipeline file that just emits `hello world` or something); once it gets merged, we can trigger builds from this PR to get validation about the work here.

Reply:
Separated in https://github.com/elastic/logstash-filter-elastic_integration/pull/125. Since I don't have concerns about pipeline definition (which is similar to we have with other pipelines), I have included Plugin build scripts to separate from this BIG change to focus only E2E tests.